src/ripple/app/misc/impl/AmendmentTable.cpp

```diff
@@ -239,8 +242,13 @@ class AmendmentTableImpl final : public AmendmentTable
         AmendmentState const& state,
         std::lock_guard<std::mutex> const& sl) const;
 
+    void
+    persistVote(uint256 const& amendment, std::string const& name, bool vote)
```

Comment:
My intuition tells me that a variable named `vote` is going to tell me whether I should vote _for_ a thing. This would be significantly clearer if `vote` was renamed `vetoed` or `voteVeto` or something like that.

Reply:
Done.